_posts/en/newsletters/2025-02-14-newsletter.md

```diff
@@ -0,0 +1,262 @@
+  received orphan transaction is evicted.
+
+  0xB10C found that, on some days, more than 10 million oprhans were
+  evicted by his node, with a peak rate of over 100,000 evictions per
+  minute.  Upon investigating, he found ">99% of these [...] are similar
+  to this [transaction][runestone tx], which seems related to runestone
+  mints [a colored coin (NFT) protocol]"  It appeared that many of the
+  same orphan transactions were repeatedly requested, randomly evicted a
+  short time later, and then requested again.
+
+- **Updated proposal for updated BIP process:** Mark "Murch" Erhardt
+  [posted][erhardt bip3] to the mailing list to announce that his draft
+  BIP for a revised BIP process has been assigned the identifier BIP3
+  and is ready for additional review---possibly its last round of review
+  before being merged and activated.  Anyone with opinions is encouraged
+  to leave feedback on the [pull request][bips #1712].
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[Cluster mempool: introduce TxGraph][review club 31363] is a PR by
+[sipa][gh sipa] that introduces the `TxGraph` class, which encapsulates
+knowledge about the (effective) fees, sizes, and dependencies between
+all mempool transactions, but nothing else. It is part of the [cluster
+mempool][topic cluster mempool] project and brings a comprehensive
+interface that allows interaction with the mempool graph through
+mutation, inspector, and staging functions.
+
+Notably, `TxGraph` does not have any knowledge about `CTransaction`,
+inputs, outputs, txids, wtxids, prioritization, validity, policy
+rules, and a lot more. This makes it easier to (almost) fully specify
+the class's behavior, allowing for simulation-based tests - which are
+included in the PR.
+
+
+{% include functions/details-list.md
+  q0="What is the mempool \"graph\" and to what extent does it exist in
+  the mempool code on master?"
+  a0="On master, the mempool graph exists implicitly as the set of
+  `CTxMemPoolEntry` objects as nodes, and their ancestor/dependant
+  relationships which can be recursively traversed with
+  `GetMemPoolParents()` and `GetMemPoolChildren()`."
+  a0link="https://bitcoincore.reviews/31363#l-26"
+
+  q1="What are the benefits of having a `TxGraph`, in your own words?
+  Can you think of drawbacks?"
+  a1="Benefits include: 1) `TxGraph` enables the implementation of
+  [cluster mempool][topic cluster mempool], with all of its benefits. 2)
+  Better encapsulation of mempool code, using more efficient data
+  structures. 3) Makes it easier to interface with and reason about the
+  mempool, abstracting away topological details such as not
+  double-counting replacements.
+
+  Drawbacks include: 1) The significant review and testing efforts
```

Comment:
Not sure if this newline is the reason that the rendering of this is incorrect, so a tentative suggestion to fix it

```suggestion
  double-counting replacements. Drawbacks include: 1) The significant review and testing efforts
```